feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -101,29 +99,36 @@ public void start(Stage stage) {
      * Fetch feature collection from portal and load as feature layer.
      */
     private void fetchFeatureCollectionFromPortal() {
+
         if (!inputTextField.getText().isEmpty()) {
             // crate portal item
             PortalItem portalItem = new PortalItem(portal, inputTextField.getText());
-                // create feature collection and add to the map as a layer
-                featureCollection = new FeatureCollection(portalItem);
-                FeatureCollectionLayer  featureCollectionLayer = new FeatureCollectionLayer(featureCollection);
-                // add feature layer to the map
-                map.getOperationalLayers().clear();
-                map.getOperationalLayers().add(featureCollectionLayer);
-
-                featureCollectionLayer.addDoneLoadingListener(() -> {
-                    if(portalItem.getLoadStatus()== LoadStatus.LOADED) {
-                        if (portalItem.getType() != PortalItem.Type.FEATURE_COLLECTION) {
-                            new Alert(Alert.AlertType.ERROR, "This is not valid Feature Collection.").show();
-                        }
+            portalItem.loadAsync();
+
+            portalItem.addDoneLoadingListener(()->{
```

Comment:
Just a bit of formatting here:
```suggestion
            portalItem.addDoneLoadingListener(() -> {
```